### PR TITLE
[PS|SS] TAV-282: Remove county from responses

### PR DESCRIPTION
**Pull request by ssaleem-ee**

### CODE REVIEW

File: app/address/osgb/DbAddress.scala [hunk 1]
```diff
@@ -42,7 +42,7 @@ trait Document {
 case class DbAddress(
                       id: String,
                       lines: List[String],
-                      town: Option[String],
+                      town: String,
```
Comment (Ardesco) [inline app/address/osgb/DbAddress.scala:45]: Is this generic cleanup?  Seems out of place in a PR to remove country.

File: app/osgb/services/Counties2.scala [hunk 2]
```diff
@@ -14,30 +14,46 @@
...
+//        if (!line(0).startsWith("LOCAL")) {
+//          val localCustodianCode = line(0)
+//          val county = line(2)
+//          data(localCustodianCode) = Capitalisation.normaliseAddressLine(county)
+//        }
+//      }
+//    }
+//    new Counties2(data.toMap)
+//  }
+//}
```
Comment (Ardesco) [inline app/osgb/services/Counties2.scala:59]: Should this file be deleted?

**Review by Ardesco — CHANGES_REQUESTED** [review]

File: app/address/osgb/DbAddress.scala (hunk 1 above)
Comment (ssaleem-ee) [reply app/address/osgb/DbAddress.scala:45]: Yeh - general cleanup.

File: app/osgb/services/Counties2.scala (hunk 2 above)
Comment (ssaleem-ee) [reply app/osgb/services/Counties2.scala:59]: yes - let me do that now.

Comment (ssaleem-ee) [reply app/osgb/services/Counties2.scala]: done

**Review by Ardesco — APPROVED** [review]

**Pull request merged**